docs/usage/models.md

```diff
@@ -165,16 +149,16 @@ print(m.model_dump())
 
 For self-referencing models, see [postponed annotations](postponed_annotations.md#self-referencing-models).
 
-## "From Attributes" (aka ORM Mode/Arbitrary Class Instances)
+## Arbitrary class instances
 
-Pydantic models can be created from arbitrary class instances to support models that map to ORM objects.
+(Formerly known as "ORM Mode", or building models from attributes.)
 
-To do this:
+Pydantic models can be created from arbitrary class instances to support models that map to ORM objects.
 
-1. The [Config](model_config.md) property `from_attributes` must be set to `True`.
-2. The special constructor `from_orm` must be used to create the model instance.
+To do this, use the `model_config` property on the model with `from_attributes` set to `True`. See
+[Model Config](model_config.md) and [ConfigDict](../api/config.md) for more information.
 
-The example here uses SQLAlchemy, but the same approach should work for any ORM.
+The example here uses [SQLAlchemy](https://www.sqlalchemy.org/), but the same approach should work for any ORM.
```

Comment:
For what it's worth, I think we should focus less intently on ORM here — I would phrase it as "Pydantic models can also be created from arbitrary class instances by reading the instance attributes corresponding to the model field names. One common application of this functionality is integration with ORMs." or similar; right now it seems like this functionality is specific to ORMs but it's actually much more general, and I think with the recent name changes, imo, it's worth introducing it in that way.